### ISSUE TITLE
Text Corruption when being warped between servers

### ISSUE THREAD

**Issue by Julienraptor01** [IrisShaders/Iris#1831]
### What happened?

it look like the text that appear when being warped to another server while on a server get corrupted

### Screenshots

![image](https://user-images.githubusercontent.com/49841713/210759654-558532fd-6428-453d-bc5e-92e5afda6e38.png)

### Log output

[latest.log](https://github.com/IrisShaders/Iris/files/10351624/latest.log)

### Minecraft Version

Minecraft 1.19.2

### Iris Version

iris-mc1.19.2-1.5.0.jar

### Operating System

Windows 10

### What is your GPU?

NVIDIA GeForce RTX 2060 Mobile

### Additional context

steps to reproduce (why isn't there a thing for this in the template ?)
go on hypixel.net
do /lobby skyblock
open the right click game menu
and right click on the planet to get warped to skyblock
look at the text which appear on the loading screen when warping

i used FO 4.5.6 so there could me a mod incompatibility that caused it

**Comment by Timolino456** [comment]
Well, first step would be trying to reproduce it with only Sodium and Iris. If it doesn't happen anymore then, enable mods until it happens again and you find the offending mod. Then disable Iris again and see if it still happens. If no, it's an incompatibility, if yes it's just the other mod.

**Comment by Julienraptor01** [comment]
> Well, first step would be trying to reproduce it with only Sodium and Iris. If it doesn't happen anymore then, enable mods until it happens again and you find the offending mod. Then disable Iris again and see if it still happens. If no, it's an incompatibility, if yes it's just the other mod.

Yeah i'll try with only iris + it's required dependancy then do the binary search way to find which mod cause the problem but i didn't have time to do it rn

**Comment by Lolothepro** [comment]
need confirmation that this bug has not been fixed.
